.github/copilot-instructions.md

```diff
@@ -0,0 +1,90 @@
+# Copilot Instructions for Sous Chefs Cookbooks
+
+## Repository Overview
+
+**Chef cookbook** for Percona MySQL Server, client tools, XtraBackup, Toolkit, and XtraDB Cluster. Representative of Sous Chefs cookbook ecosystem.
+
+**Key Facts:** Ruby-based, ~800KB, 46 Ruby files, Chef >= 16 required, supports CentOS 7+/Debian 10+/Ubuntu 18.04+
+
+## Project Structure
+
+**Critical Paths:**
+- `recipes/` - 11 Chef recipes (default, server, client, backup, cluster, etc.)
```

Comment:
This needs to be generalized. These are for recipes that the cookbook provides. If this is a resource driven cookbook, they may not be any recipes or very few which uses the resources.